modules/core/arrow-data/build.gradle

```diff
@@ -12,6 +12,7 @@ dependencies {
     testCompile project(':arrow-test')
     testCompile project(':arrow-free')
     testCompile project(':arrow-instances-free')
+    testCompile project(':arrow-data-instances-effects')
```

Comment:
This is required since tests are all on this module but this particular instance is in the newly added module.